packages/SwingSet/test/vat-transcript-maybe.js

```diff
@@ -0,0 +1,18 @@
+export default function setup(syscall, _state, _helpers, vatPowers) {
+  let ephemeralCounter = 0;
+
+  function dispatch(_vatDeliverObject) {
+    ephemeralCounter += 1;
```

Comment:
You might want to limit this to the case where `vatDeliverObject[0] === 'send'`, because when the GC work lands, you may get extra messages showing up (specifically a `dropExports` when nobody uses the root object we create for this vat). I don't remember if there's something else that might keep the root alive, inhibiting that message, but it might be good to be insensitive to it anyways.